src/pages/Facility/settings/devices/DeviceDetail.tsx

```diff
@@ -144,6 +145,23 @@ export default function DeviceDetail({ facilityId, deviceId }: Props) {
       <div className="flex items-center justify-between">
         <PageTitle title={device.registered_name} />
         <div className="flex items-center gap-2">
+          <LinkDepartmentsSheet
+            entityType="device"
+            entityId={deviceId}
+            facilityId={facilityId}
+            currentOrganizations={[device.managing_organization]}
```

Comment:
Alright I see...let's modify this instead, something like:

```suggestion
            currentOrganizations={device.managing_organization ? [device.managing_organization] : []}
```